src/content/docs/usage/workspaces.mdx

```diff
@@ -29,18 +39,25 @@ You can create a Workspace.
    daytona create
    ```
 
-2. Select the Git Provider and repository you want your Workspace to be configured with.
-3. Enter a name for your Workspace.
+2. Select the Git Provider and the repository you want your Workspace to be configured with. This selection links your Workspace directly to your version control system.
+
+   ```text
+      Choose a Provider
+      2 items
+      ===
+      GitHub
+      Enter a custom repository URL
+   ```
+
+3. Enter a name for your Workspace. Provide a descriptive name for your Workspace that reflects its purpose or project.
```

Comment:
@stefanicjuraj When you test daytona you will see that it creates a name for you using the info from the repo, but you can edit it to be "descriptive name for your Workspace that reflects its purpose or project"